lib/grant-types/password-grant-type.js

```diff
@@ -88,7 +88,7 @@ PasswordGrantType.prototype.getUser = function(request) {
     throw new InvalidRequestError('Invalid parameter: `password`');
   }
 
-  return promisify(this.model.getUser, 2)(request.body.username, request.body.password)
+  return promisify(this.model.getUser, 2)(request.body.username, request.body.password, request)
```

Comment:
The second argument to `promisify` reflects the number of arguments the function takes.

```
return promisify(this.model.getUser, 3)(request.body.username, request.body.password, request)
//                                   ^
```